### ISSUE TITLE
Blender 3.3.1 material color set incorrectly

### ISSUE THREAD

**Issue by danpf** [BradyAJohnston/MolecularNodes#98]
Hello,
https://github.com/BradyAJohnston/MolecularNodes/blob/7648a3f0c28d347d8c17d99532dc3f24b218e0ef/nodes.py#L43

something is set wrong at this block:
```
        mat = bpy.data.materials.new('MOL_atomic_material')
        mat.use_nodes = True
        node_att = mat.node_tree.nodes.new("ShaderNodeAttribute")
        node_att.attribute_name = "Color"
        node_att.location = [-300, 200]
        mat.node_tree.links.new(node_att.outputs['Color'], mat.node_tree.nodes['Principled BSDF'].inputs['Base Color'])
```

in Blender 3.3.1 I see 
<img width="1096" alt="image" src="https://user-images.githubusercontent.com/16297104/205163637-df037f7e-2124-4386-80a7-1cd2295cccfa.png">
note the undefined

then changing input to:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/16297104/205163706-49ac4429-2534-45ee-a3bd-0746ce593cb8.png">
and putting in the Name 'Color' it gets fixed.

<img width="926" alt="image" src="https://user-images.githubusercontent.com/16297104/205163756-5aed955e-63d2-452f-a9aa-20df0b41c7d1.png">

I'm not sure how to fix this in the python api at first glance though.

**Comment by danpf** [comment]
sorry - just an update. There is no problem with the code. It is in the base .blend file.

when I open it and go to the Shading tab i see this:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/16297104/205244072-bfe29d5c-647b-4fd7-9bd8-c73c95977612.png">

**Comment by BradyAJohnston** [comment]
Hi @danpf ! Thanks for opening an issue. The development version of Molecular Nodes is only compatible with Blender 3.4+, and the red node that you are seeing is because you loaded it in an earlier version of Blender. I will make this more clear in the readme. 

The earlier released versions of Molecular Nodes (v0.13) are compatible with Blender 3.3+
